datasets/NJ_LiDAR.yaml

```diff
@@ -0,0 +1,22 @@
+Name:   New Jersey Statewide LiDAR 
+Description:  Elevation datasets in New Jersey have been collected over several years as several discrete projects.  Each project covers a geographic area, which is a subsection of the entire state, and has differing specifications based on the available technology at the time and project budget.  The geographic extent of one project may overlap that of a neighboring project.  Each of the 18 projects contains deliverable products such as:
+•	LAS (Lidar point cloud) files, unclassified/classified, tiled to cover project area
+•	Relevant metadata records or documents, most adhering to the Federal Geographic Data Committee’s (FGDC) Content Standard for Digital Geospatial Metadata (CSDGM).
+•	Tiling index feature class or shapefile
+•	Flights lines feature class or shapefile
+•	Digital Elevation Model in image format or Esri grid format
+•	Other derivative data products such as contour lines feature class or shapefile.
+Contact:  njgin@oit.nj.gov
+ManagedBy:  The New Jersey Office of GIS, NJ Office of Information Technology
+UpdateFrequency: None
+Tags:
+  - lidar
+  - elevation
+  - geospatial
+  - mapping
+License:  The State of New Jersey provides the service "as is". The State makes no guarantee or warranty concerning the accuracy of information contained in the data. Also, the State makes no warranty, either expressed or implied, regarding the condition of the product or its fitness for any particular purpose. The burden for determining fitness for use lies entirely with the user. Although these data files and service have been processed successfully on computers at the State, no warranty is made by the State regarding the use of the data or service on any other system, nor does the fact of distribution constitute or imply such a warranty.
+Resources:
+  - Description: https://njgin.nj.gov/njgin/edata/elevation

```

Comment:
Please change to a text description of the bucket contents. Can add link if needed, but should also be a brief description here.